devenv/scripts/make_e2e.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+make e2e
```

Comment:
is this file needed?

Reply:
I couldn't get the docker image to run the command without calling it from within a shell script. I can try to make it work without it if you'd like, but I spent a while and kept getting a weird "exec" error